src/lib.rs

```diff
@@ -1,30 +1,16 @@
-use hyper::{
-    http::header::{HeaderName, HeaderValue},
-    http::request::Parts,
-    http::uri::Scheme,
-    Body, Request, Response, StatusCode,
-};
+use crate::httputil::*;
```

Comment:
nit: Would `http_util` be more appropriate?

Reply:
Probably. I let a Go-ism slip in there.